spec/spec_helper.rb

```diff
@@ -29,7 +29,6 @@
 require 'mail'
 require 'percy'
 require 'rspec_performance_summary'
-require 'super_diff/rspec-rails'
```

Comment:
PR to make `super_diff` compatible with Ruby 3: https://github.com/mcmire/super_diff/pull/ 118